### DISCUSSION TITLE
How to use CSS variable autocompletion

### DISCUSSION THREAD

**Discussion by lucaskfp** [argyleink/open-props#254] [Q&A]
I'm developing a project with NextJs but I couldn't configure autocomplete for open-props

**Comment by argyleink** [comment]
the vscode autocomplete plugins have been acting funny.. 

curious, are you able to getting autocomplete working in html/js but not css? that's what's happening to most folks.

**Reply by lucaskfp** [reply]
Funny a css plugin that doesn't work with css 🤔

**Comment by argyleink** [comment]
The maintainer found us and helped us out https://github.com/argyleink/open-props/pull/256 
give it another shot?!
